### PR TITLE
Addition of Fogflow Performace Document

### PR DESCRIPTION
**Pull request by pradumnnec**

### CODE REVIEW

**Review by showersky — APPROVED** [review]

File: doc/en/source/performanceNew.rst [hunk 1]
```diff
@@ -24,91 +24,266 @@ Throughput and latency to create new entities
 --------------------------------------------------
```
Comment (naveennec) [inline doc/en/source/performanceNew.rst:24]: in line no 15- just tell only FogFlow deployment, no need to explain "to test the performance"
in line no 17- don't use personal world, better to write "Listner client is a VM is deployed in VM having 4cpu, 8GB RAM with functionality  to measure the "time delay of context update from the moment sent by a context producer to the time received by a subscriber"
in line no 19- better to write Apache Jmeter tool is used  to produce input for fogflow broker.

File: doc/en/source/performanceNew.rst [hunk 2]
```diff
@@ -24,91 +24,266 @@ Throughput and latency to create new entities
...
 
 .. figure:: figures/imgUpsertCreate.png
 
 .. figure:: figures/imgUpsertCreate2.png
 
-**update**
+.. figure:: figures/createdata.png
+
+**Analysis of Graphs**:
+The above graphs are plotted against Create API in Fogflow. The Y axis in graph represents response time and X axis represents timestamp to gnerate real time environment. The line parallel to X axis is the mean of response time for all the entities created using Create API. The graph shows the response time in contrast to each and every request that is handled by Fogflow broker.
```
Comment (naveennec) [inline doc/en/source/performanceNew.rst:36]: this heading should be above the graph in all graphs then paste the graph, then tell about the result table and then paste the table.

**Review by naveennec — CHANGES_REQUESTED** [review]

**Comment by pradumnnec** [comment]
Incorporated Comments

File: doc/en/source/performanceNew.rst [hunk 3]
```diff
@@ -12,103 +12,326 @@ This tutorial introduces the performace evaluation of NGSILD based context manag
...
 
 
 Throughput and latency to create new entities
 --------------------------------------------------
 
 **Create**
+************
+Fogflow allows the creation of entities either in v1 format or NGSI-LD format. These requests are made to Fogflow broker and in return Fogflow broker responds with either successful entity creation result or with any error if associated. Thus it forms the main backbone of Fogflow architectue that allows data creation.
 
 .. figure:: figures/imgUpsertCreate.png
```
Comment (naveennec) [inline doc/en/source/performanceNew.rst:31]: Name should be added below each diagram like "Fig:1- 20 Threads running ach 200 seq request"

**Review by naveennec — CHANGES_REQUESTED** [review]

**Comment by pradumnnec** [comment]
Incorporated Comments

File: doc/en/source/performanceNew.rst (hunk 1 above)
Comment (pradumnnec) [reply doc/en/source/performanceNew.rst]: Incorporated Comments

File: doc/en/source/performanceNew.rst (hunk 2 above)
Comment (pradumnnec) [reply doc/en/source/performanceNew.rst:39]: Incorporated Comments

File: doc/en/source/performanceNew.rst [hunk 4]
```diff
@@ -12,103 +12,326 @@ This tutorial introduces the performace evaluation of NGSILD based context manag
 Experiement Setup
 -------------------
 
-**FogFlow system:** To test the performance, FogFlow is deployed on one cloud node(4vCPUs, 16 GB RAM) and two edge node(4vCPUs, 8 GB RAM) in doker container.
+**FogFlow system:** FogFlow is deployed on one cloud node(4vCPUs, 16 GB RAM) and two edge node(4vCPUs, 8 GB RAM) in doker container.
 
-**Listener client:** To measure the delay of context update from the moment sent by a context producer to the time received by a subscriber we are using listener client. Listener client is deployed on a VM(4cpu, 8GB RAM)
```
[the remark this answers is not in the document]
Comment (pradumnnec) [reply doc/en/source/performanceNew.rst:17]: Incorporated Comments

File: doc/en/source/performanceNew.rst (hunk 3 above)
Comment (pradumnnec) [reply doc/en/source/performanceNew.rst:31]: Incorporated Comments

**Pull request merged**